wire-library/wire-runtime/build.gradle.kts

```diff
@@ -43,6 +43,7 @@ kotlin {
       languageSettings.useExperimentalAnnotation("kotlin.Experimental")
     }
     val commonMain by getting {
+      kotlin.srcDir("src/generated/kotlin")
```

Comment:
The right way to do this is to create a task which has an output (like [this](https://github.com/cashapp/zipline/blob/85116a14c55dd2ed28138e1f483b43224943978f/zipline/build.gradle.kts#L17-L37)) and add the task to the source set (like [this](https://github.com/cashapp/zipline/blob/85116a14c55dd2ed28138e1f483b43224943978f/zipline/build.gradle.kts#L84)).